app_legacy/Helpers/render/game.php

```diff
@@ -190,6 +190,53 @@ function renderGameCard(int|array $game): string
     return $tooltip;
 }
 
+function RenderGameSort(bool $isFullyFeaturedGame, ?int $flags, int $officialFlag, int $gameID, ?int $sortBy): void
+{
+    echo "<div class='py-3'><span>";
+    echo "Sort: ";
+
+    $flagParam = ($flags != $officialFlag) ? "f=$flags" : '';
+
+    $sortType = ($sortBy < 10) ? "^" : "<sup>v</sup>";
+    // Used for options which sort in Descending order on first click
+    $sortReverseType = ($sortBy >= 10) ? "^" : "<sup>v</sup>";
+
+    $sort1 = ($sortBy == 1) ? 11 : 1;
+    $sort2 = ($sortBy == 2) ? 12 : 2;
+    $sort3 = ($sortBy == 3) ? 13 : 3;
+    $sort4 = ($sortBy == 4) ? 14 : 4;
+    $sort5 = ($sortBy == 5) ? 15 : 5;
+
+    $mark1 = ($sortBy % 10 == 1) ? "&nbsp;$sortType" : "";
+    $mark2 = ($sortBy % 10 == 2) ? "&nbsp;$sortType" : "";
+    $mark3 = ($sortBy % 10 == 3) ? "&nbsp;$sortType" : "";
+    $mark4 = ($sortBy % 10 == 4) ? "&nbsp;$sortType" : "";
+    $mark5 = ($sortBy % 10 == 5) ? "&nbsp;$sortType" : "";
+
+    $reverseMark1 = ($sortBy % 10 == 1) ? "&nbsp;$sortType" : "";
+    $reverseMark2 = ($sortBy % 10 == 2) ? "&nbsp;$sortType" : "";
+    $reverseMark3 = ($sortBy % 10 == 3) ? "&nbsp;$sortType" : "";
+    $reverseMark4 = ($sortBy % 10 == 4) ? "&nbsp;$sortType" : "";
+    $reverseMark5 = ($sortBy % 10 == 5) ? "&nbsp;$sortType" : "";
```

Comment:
This logic is identical to `markX`. Therefore it's not actually reversing anything. I suspect you want to replace`$sortType` with `$sortReverseType`.

Reply:
LOL thank you, updated! When I was testing I was confusing myself which way was which 💀